crates/asm/txs/bridge-v1/src/errors.rs

```diff
@@ -95,3 +96,25 @@ pub enum WithdrawalParseError {
     #[error("Transaction is missing output that fulfilled user withdrawal request")]
     MissingUserFulfillmentOutput,
 }
+
+/// Errors that can occur when parsing cooperative transactions.
+///
+/// When these parsing errors occur, they are logged and the transaction is skipped.
+/// No further processing is performed on transactions that fail to parse.
+#[derive(Debug, Error)]
+pub enum CooperativeParseError {
+    /// The auxiliary data in the withdrawal fulfillment transaction doesn't have correct length.
+    #[error("Invalid auxiliary data: expected {COOPERATIVE_TX_AUX_DATA_LEN} bytes, got {0} bytes")]
+    InvalidAuxiliaryData(usize),
+
+    /// The transaction type byte in the tag does not match the expected withdrawal fulfillment
+    /// transaction type.
+    #[error("Invalid transaction type: expected type to be {COOPERATIVE_TX_TYPE}, got {0}")]
+    InvalidTxType(TxType),
+
+    #[error("Transaction is missing bridge input")]
+    MissingBridgeInput,
+
+    #[error("Transaction is missing output that fulfilled user withdrawal request")]
+    MissingWithdrawalOutput,
```

Comment:
Please add docstrings here. They help out when using these variants as a developer because we don't see the `#error` macro in the LSP but the docstrings we do.